buildkite/resource_pipeline.go

```diff
@@ -622,9 +622,9 @@ func (*pipelineResource) Schema(ctx context.Context, req resource.SchemaRequest,
 						Computed: true,
 						Optional: true,
 						MarkdownDescription: "Whether to filter builds to only run when the condition in `filter_condition` is true." +
-							"Only valid for GitHub repositories when `trigger_mode = \"code\"`.",
+							"Only valid for Bitbucket, or GitHub repositories when `trigger_mode = \"code\"`.",
 						PlanModifiers: []planmodifier.Bool{
-							custom_modifier.WhenRepositoryProviderIs(custom_modifier.RepositoryProviderGitHub),
+							custom_modifier.WhenRepositoryProviderIs(custom_modifier.RepositoryProviderGitHub, custom_modifier.RepositoryProviderBitbucket),
```

Comment:
Do we have a test for this?

Reply:
No, because I wasn't sure of the value in terms of detecting a regression in the code. Since its so specific to the API implementation. What do you think? Should I add one? @mcncl

Reply:
I'm indifferent to it; we already have a test for Git which effectively checks the same things.